src/bot/conversation/authorization.py

```diff
@@ -3,13 +3,13 @@
 from telegram import InlineKeyboardButton, InlineKeyboardMarkup, Update
 from telegram.ext import CallbackContext, CallbackQueryHandler, CommandHandler, ConversationHandler
 
-from constants import callback_data, states
-from conversation.menu import menu_conversation
-from conversation.timezone import ASK_FLAG, set_timezone_from_keyboard, timezone_conversation
+from bot.constants import callback_data, states
+from bot.conversation.menu import menu_conversation
+from bot.conversation.timezone import ASK_FLAG, set_timezone_from_keyboard, timezone_conversation
+from bot.decorators.logger import async_error_logger
+from bot.menu_button import COMMANDS_UNAUTHORIZED, menu_button
```

Comment:
Понимаю что код не твой, но давай сразу по чуть-чуть его улучшать
Давай сделаем такой импорт, это добавит читаемости. 
```suggestion
from bot.conversation import timezone
from bot.decorators.logger import async_error_logger
from bot import menu_button
```